AISKU/Tests/applicationinsights.e2e.tests.ts

```diff
@@ -153,11 +153,12 @@ export class ApplicationInsightsTests extends TestClass {
             steps: [() => {
                 this._ai.trackEvent({ name: 'event', properties: { "prop1": "value1" }, measurements: { "measurement1": 200 } });
             }].concat(this.asserts(1)).concat(() => {
-
                 const payloadStr: string[] = this.getPayloadMessages(this.successSpy);
                 if (payloadStr.length > 0) {
                     const payload = JSON.parse(payloadStr[0]);
                     const data = payload.data;
+                    Assert.ok( payload && payload.iKey);
```

Comment:
this will check the config iKey is used when iKey is not defined in trackevent()